README.md

```diff
@@ -251,6 +251,9 @@ npm run build:debug
 
 With these steps completed, you will be able to debug C++ files directly within Chrome DevTools.
 
+This version runs in single-threaded mode. We also offer a multithreaded version, distinguished by
+":mt".
+
```

Comment:
The above commands will build and run a single-threaded version. To build a multi-threaded version, just add the suffix ":mt" to each command. For example:

```
npm run build:mt
npm run build:mt:debug
npm run serser:mt
```